docs/source/operations/install_instantiate_chaincode.md

```diff
@@ -48,13 +48,14 @@ network:
 ```
 
 <a name = "run_network"></a>
-## Running playbook to install and instantiate chaincode in BAF deployed Hyperledger Fabric network
+## Chaincode Operations in BAF for the deployed Hyperledger Fabric network
 
-The playbook [chaincode-install-instantiate.yaml](https://github.com/hyperledger-labs/blockchain-automation-framework/tree/master/platforms/hyperledger-fabric/configuration/chaincode-install-instantiate.yaml) is used to install and instantiate chaincode for the existing fabric network.
+The playbook [chaincode-ops.yaml](https://github.com/hyperledger-labs/blockchain-automation-framework/tree/master/platforms/hyperledger-fabric/configuration/chaincode-ops.yaml) is used to install and instantiate chaincode for the existing fabric network.
+For Fabric v2.2 multiple operations such as approve, commit and invoke the chaincode are available in the same yaml file. 

```

Comment:
This line should be changed to
For Fabric v2.2 multiple operations such as approve, commit and invoke the chaincode are available in the same playbook.